e2e/integration/wordpress-01/__snapshots__/yoast-package.spec.js.snap

```diff
@@ -876,3 +877,51 @@ exports[`Yoast Package > Custom Taxonomy > should render the schema tag #0`] = `
   }
 </script>
 `;
+
+exports[`Yoast Package > Post > should render the Twitter tags #1`] = `
+<meta data-rh="true" name="twitter:label1" content="Written by" />
+`;
+
+exports[`Yoast Package > Post > should render the Twitter tags #2`] = `
+<meta data-rh="true" name="twitter:data1" content="admin" />
+`;
+
+exports[`Yoast Package > Post > should render the Twitter tags #3`] = `
+<meta data-rh="true" name="twitter:label2" content="Est. reading time" />
+`;
+
+exports[`Yoast Package > Post > should render the Twitter tags #4`] = `
+<meta data-rh="true" name="twitter:data2" content="0 minutes" />
+`;
+
+exports[`Yoast Package > Page > should render the Twitter tags #1`] = `
+<meta data-rh="true" name="twitter:label1" content="Written by" />
+`;
+
+exports[`Yoast Package > Page > should render the Twitter tags #2`] = `
+<meta data-rh="true" name="twitter:data1" content="admin" />
+`;
+
+exports[`Yoast Package > Page > should render the Twitter tags #3`] = `
+<meta data-rh="true" name="twitter:label2" content="Est. reading time" />
+`;
+
+exports[`Yoast Package > Page > should render the Twitter tags #4`] = `
+<meta data-rh="true" name="twitter:data2" content="1 minute" />
+`;
+
+exports[`Yoast Package > CPT > should render the Twitter tags #1`] = `
+<meta data-rh="true" name="twitter:label1" content="Written by" />
+`;
+
+exports[`Yoast Package > CPT > should render the Twitter tags #2`] = `
+<meta data-rh="true" name="twitter:data1" content="admin" />
+`;
+
+exports[`Yoast Package > CPT > should render the Twitter tags #3`] = `
+<meta data-rh="true" name="twitter:label2" content="Est. reading time" />
+`;
+
+exports[`Yoast Package > CPT > should render the Twitter tags #4`] = `
+<meta data-rh="true" name="twitter:data2" content="0 minutes" />
+`;
```

Comment:
Those snapshots were missing, but because by default the cypress snapshot plugin runs with

```js
{ 
  "autopassNewSnapshots": true,    // Automatically save & pass new/non-existing snapshots
}
```

It looks like those snapshots were never generated and as a result the tests never failed on the CI.